e2e/cypress/integration/at_mentions/at_mentions_spec.js

```diff
@@ -126,6 +126,12 @@ describe('at-mention', () => {
 
         const body = `@${sender.username}: ${message}`;
 
+        cy.get('@notifySpy').should('have.been.calledWithMatch', 'Town Square', (args) => {
+            expect(args.body, `Notification body: "${args.body}" should match: "${body}"`).to.equal(body);
+            expect(args.tag, `Notification tag: "${args.tag}" should match: "${body}"`).to.equal(body);
+            return true;
```

Comment:
There's a recent change on second parameters, probably due to browser.  So, instead of matching exactly the object properties, only assert the body and tag which are the most important properties to check.